doc/rm/ref_manual_fpga.md

```diff
@@ -1,80 +0,0 @@
----
```

Comment:
why is this file deleted?

Reply:
This appears to be an error. @alexsapoznikov, I know you were having trouble building around this file.  Though it seems you've deleted in entirely. Could you please put it back?
```
cd $OT_ROOT
git checkout v1_ready
# get the original file back
git checkout master -- doc/rm/ref_manual_fpga.md
# Add it back into your PR
git add doc/rm/ref_manual_fpga.md
git commit
git push
```
when you next push this file should be added back (and unchanged)

Reply:
The file ref_manual_fpga.sv is supposed to be in  ./doc/ug/ref_manual_fpga.md and it originally was in this directory. When I debugged the failure to build a document, the error message reported that the ref_manual_fpga.md file could not be found in **./doc/rm directory**, and I copied this file to ./doc/rm directory. This did not solve a problem and I deleted this file.